lib/recommendation-server.js

```diff
@@ -27,7 +27,7 @@ function RecommendationServer(opts) {
   this.timeout = opts.timeout;
   // Note: this is the xhr service, not an xhr instance
   this.xhr = opts.xhr;
-  this.console = opts.console;
+  this.logger = new opts.Logger('server', opts.win);
```

Comment:
Also, let's go with upper camel case for the name of the class in the logger prefix: `new opts.Logger('RecommendationServer', ... `. Please replace this throughout